content/projects/katas/level-1/_index.md

```diff
@@ -177,4 +177,31 @@ eg `combine([11,22,33], [1,2,3])` should return
 
 ```
 [11,1,22,2,33,3]
-```
\ No newline at end of file
+```
+
+## Instructions for Reviewers
+
+- Ensure that every task has its own file, and branch.
+- Check if a specific task wants an output to be returned, if so the **return** keyword should be used to stop the execution of the function.
```

Comment:
RE: 184, 185, & 186
Does this even need to be said at all? I think this is in the project instructions above as well? Things should be DRY